src/core/mantle/types/expression.ts

```diff
@@ -1,3 +1,7 @@
+export type ExpressionContainer = {
+  expression: StyleExpression;
+};
+
 export type StyleExpression = ConditionsExpression | string | boolean | number;
```

Comment:
```suggestion
export type StyleExpression = ConditionsExpression | string;
```

Always an expression should be described as a string?

Reply:
Yeah, To enforce the expression behaviour.
Though I went with 
```
{
  marker: {
    pointSize: 23,
  }
}
```

```
{
 marker: {
   pointSize: {
     expression: 23,
     }
   }
}
```
Both valid statements.

Reply:
Oh ok, but any type is settable as a value (`T[K] | StyleExpression`). so to disambiguate, it would be better to allow only expressions to be written inside EXPRESSION. If you want to embed a value directly, write it without expression.

Reply:
makes sense, done.